package.json

```
@@ -77,6 +77,7 @@
     "react-outside-click-handler": "^1.3.0",
     "reading-time": "^1.5.0",
     "rehype-slug": "^6.0.0",
+    "remark-gfm": "^4.0.0",
```

Hello @probberechts, first of all thanks a lot for taking your time raising this PR. I ran ur branch on my local and got this error on my test site. 

```Failed to compile the file "<md file path>". Original error message:

Cannot read properties of undefined (reading 'inTable')```

Looking further into it, I realsed it could be an [issue with the latest remark-gfm plugin](https://github.com/contentlayerdev/contentlayer/issues/558#issuecomment-1736215198). As the linked comment suggests, downgrading the plugin to `3.0.1` fixed the issue for me. But didn't get much time to look deep into what has gone wrong.

```suggestion
    "remark-gfm": "^3.0.1",
```

---

Hmm, yes. It seems you only get this error with markdown tables. I didn't have any, so I didn't notice it earlier. I would be happy to downgrade to v3.0.1 since I don't see any other solution.